services/datamanager/builtin/sync/file_deletion_windows.go

```diff
@@ -0,0 +1,47 @@
+package sync
```

Comment:
does this need a go:build flag to ensure it only builds on windows?

Reply:
(same as previous file, _GOOS file suffix is doing the work)